common/source/docs/common-choosing-a-ground-station.rst

```diff
@@ -70,164 +70,190 @@ drone) and special missions for 3D mapping.
 -  `Support Forum <http://ardupilot.com/forum/viewforum.php?f=15>`__
 -  `Issues List <https://github.com/DroidPlanner/Tower/issues>`__
 -  `Source Code <https://github.com/DroidPlanner/droidplanner>`__
-
-.. note::
-
+
+.. note::
+
    Legacy versions (`DroidPlanner 2 <https://play.google.com/store/apps/details?id=org.droidplanner>`__
    and `Droid Planner 1 <https://play.google.com/store/apps/details?id=com.droidplanner>`__)
-   can also be downloaded from Google Play.
-
+   can also be downloaded from Google Play.
+   
+UgCS - Universal Ground Control Station
+---------------
+
+Universal and easy to use ground control station with a 3D interface. Supports APM, Pixhawk as well as drones from other manufacturers such as DJI, Mikrokopter and more.
+Intended for enthusiasts as well as professional users.
+
+Is capable of communicating with and controlling multiple drones simultaneously. 

```

Comment:
As a matter of style, best to have "proper sentences". Ie "It is capable" rather than "Is capable". This rule could be applied globally.